github/templates/README.md

```diff
@@ -117,6 +117,6 @@ About
 
 PROJECT_NAME is maintained by [Subvisual](http://subvisual.com).
 
-[![Subvisual](https://raw.githubusercontent.com/subvisual/guides/master/github/templates/logos/blue@4x.png)](http://subvisual.com)
+[<img alt="Subvisual logo" src="https://raw.githubusercontent.com/subvisual/guides/master/github/templates/logos/blue@8x.png" width="350px" />](https://subvisual.com)
```

Comment:
- According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attr-width), the `width` attribute must be an integer without a unit.
- If you check the resulting markup on GitHub, the `style` attribute is defined to set `max-width: 100%`. So ideally `max-width` should be set in the `style` attribute, overriding their value.
- Using the `8x` image version seems rather odd. Instinctively it seems to me to be overkill to load such a big image just for a README. Also, you changed the `subvisual_logo_with_name.png` file because that's what being used in most repos. Why not simply use that file in the template?
- Given GitHub automatically sets the image to fill the available space at most, have you considered instead just putting a `<div style="max-width: 350px">` around the image?

Reply:
Can you put HTML in the READMEs?

Reply:
Yes. Markdown spec supports plain HTML. You just can't use Markdown inside an HTML tag that way, but you can use any HTML snippet you want and it will be untouched (supposedly) in the end result.

Reply:
I know the Markdown spec supports plain HTML, I just think you can't do a lot on Github Markdown. Uh, I'll try later.

Re 8x version: new Mac and iPad screens use that, is there any way we can work around that pixelation?

Re `subvisual_logo_with_name.png` file name: because older repos, if we change that file to be 2x, 4x or even 8x, would need to be updated or end up with a HYYYUUUUGE logo in the README. Like, occupying almost 100% of the width.

Reply:
> I know the Markdown spec supports plain HTML, I just think you can't do a lot on Github Markdown. Uh, I'll try later.
> 
> Re 8x version: new Mac and iPad screens use that, is there any way we can work around that pixelation?
> 
> Re `subvisual_logo_with_name.png` file name: because older repos, if we change that file to be 2x, 4x or even 8x, would need to be updated or end up with a HYYYUUUUGE logo in the README. Like, occupying almost 100% of the width.

Is there any pixelation, or are you assuming? Just asking, because changing to a smaller logo the README assumes the new logo dimensions. So using the 1x logo in a high pixel density device should just show it smaller I reckon. Might be too small though, but I believe the 8x would be overkill unless we actually need to.

Reply:
> Is there any pixelation, or are you assuming?

I'll have to take a crack at this because this is from February and I don't quite recall what happened. I'll check in later I guess.